packages/steam/index.ts

```
@@ -0,0 +1,149 @@
+import * as http from 'http';
+import * as qrcode from 'qrcode-terminal';
+
+class Steam {
+    httpClient: http.ClientRequest | null = null;
+    socketPath: string = '/tmp/steam.sock';
+
+    constructor() { }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)